enarx-keep-sev-shim/src/gdt.rs

```diff
@@ -18,8 +18,89 @@ use x86_64::{PrivilegeLevel, VirtAddr};
 const STACK_NUM_PAGES: usize = 5;
 static mut STACKS: [[Page; STACK_NUM_PAGES]; 7] = [[Page::zeroed(); STACK_NUM_PAGES]; 7];
 
-pub(crate) static mut TSS: TaskStateSegment = TaskStateSegment::new();
-pub(crate) static mut GDT: GlobalDescriptorTable = GlobalDescriptorTable::new();
+/// FIXME: will be replaced by a dynamically allocated stack with safe guards
+pub static mut LEVEL_0_STACK: [Page; 5] = [Page::zeroed(); 5];
```

Comment:
Do we need a lock here if we have multiple CPUs?

Reply:
Not for this (will be removed anyway in the next patch).

Each CPU will get it's own GDT and TSS. So, yes, it will have a lock, then.